src/praisonai-sandbox/praisonai_sandbox/compute/e2b.py

```
@@ -0,0 +1,228 @@
+        from praisonaiagents.managed.protocols import InstanceInfo, InstanceStatus
+
+        info = self._sandboxes.get(instance_id)
+        if not info:
+            return InstanceInfo(
+                instance_id=instance_id,
+                status=InstanceStatus.STOPPED,
+                provider="e2b",
+            )
+        try:
+            running = info["sandbox"].is_running()
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)